main/idf_component.yml

```diff
@@ -1,10 +1,23 @@
 dependencies:
   # Required IDF version
   idf: ">=5.1"
-
-  mdns: "^1.1.0"
-  chmorgan/esp-libhelix-mp3: "1.0.3"
-  esp-dsp: "^1.3.4"
+  espressif/esp32-camera:
+    version: "*"
+    git: https://github.com/espressif/esp32-camera.git
+    require: public
+  espressif/esp-tflite-micro:
+    version: ">=1.2.0"
+    require: public
+  espressif/esp-dl:
+    version: ">=2.0.0"
+    path: ../components/esp-dl
+    require: public
+    rules:
+      - if: "target in [esp32s3]"
+  espressif/esp-sr:
```

Comment:
This component is already in Arduino component list from PR #8842.
As soon as the #8842 is merged, I'll remove it and make it run CI again.